src/net/sourceforge/kolmafia/request/DrinkItemRequest.java

```diff
@@ -396,12 +393,18 @@ private boolean allowBoozeConsumption() {
     return DrinkItemRequest.allowBoozeConsumption(itemName, count);
   }
 
+  private static final boolean shotglassApplies(String itemName) {
+    int inebriety = ConsumablesDatabase.getInebriety(itemName);
+    return inebriety == 1
+        && InventoryManager.getCount(ItemPool.MIME_SHOTGLASS) > 0
+        && !Preferences.getBoolean("_mimeArmyShotglassUsed")
+        && ItemDatabase.getItemId(itemName) > 0;
```

Comment:
Are you sure this is an accurate limitation? Are there drinkables other than the distillate which are 1-size and are fake items?

Reply:
That's a good question. It feels hacky to explicitly exclude the stillsuit instead of using a more general rule, but I'd be willing to bet there are some fake items that use the regular drink code for kol. I don't appear to be able to use Item.all() to test this through mafia; do you know an efficient way to check for other fake items?